src/MainTest.java

```diff
@@ -0,0 +1,96 @@
+import org.json.JSONArray;
+import org.json.JSONObject;
+import org.junit.Before;
+import org.junit.Test;
+import static org.junit.Assert.assertEquals;
+
+
+public class MainTest {
+    private JSONArray jsonArray;
+
+    @Test
+    public void testFormatDate_ValidTimestamp() {
```

Comment:
please read on DZone about test naming convention. I recommend you using:
shouldFormatDateToValidTimestamp()

p.s. in general this test is not needed because you should you LocalDateTime or ZonedDateTime